src/isa/riscv64/instr/rvcbo/exec.h

```diff
@@ -23,6 +23,13 @@
 #define CACHE_BLOCK_OPS          (CACHE_BLOCK_SIZE / CACHE_OP_SPLIT_SIZE)
 
 def_EHelper(cbo_zero) {
+  // check illegal instruction exception
+  if(!cpu.v && ((cpu.mode != MODE_M && !menvcfg->cbze) || (cpu.mode == MODE_U && !senvcfg->cbze))){
```

Comment:
<img width="399" alt="cbo-cbze" src="https://github.com/user-attachments/assets/ac315ce5-cced-4ff3-9729-0b0f0c61804f">

The CBO Spec only says it is not in M ​​mode, maybe it is in HS/HU/VS/VU mode.

Reply:
Make sense, already fix it.